app/controllers/branch_adresses_controller.rb

```diff
@@ -0,0 +1,8 @@
+class BranchAdressesController < CrudController
+
+  private
+
+  def fetch_entries
+    BranchAdress.all
+  end
```

Comment:
Do you even need this? Should call `model_scope.list` inside ListController anyways

Reply:
Yes and no :wink: 
I didn't define a `list` scope as it didn't seem necessary to me as these records should be ordered by id anyway. As I didn't have a `list` scope defined this would not have worked.
However, SQL can be a bit wonky as to which order it returns records in. Therefore it makes sense to define the `list` scope anyway and send the records ordere by `ID` just to be on the sure side.
So now that I have a `list` scope your suggestion is of course correct. :partying_face: